prisma/schema.prisma

```diff
@@ -187,7 +188,9 @@ model Exercise {
   answer           String
   testStr          String?
   explanation      String?
-  removed          Boolean?             @default(false)
+  removedAt        DateTime?
+  removedById      Int?
+  removedBy        User?                @relation("removedExercises", fields: [removedById], references: [id])
```

Comment:
Something I discovered during testing the changes with Apollo playground: `removedBy` and `flaggedBy` returns null because they refer to a different model. In order to include them in the result of querying the db for exercises, we'd have to explicitly tell Prisma to include them:

```diff
export const exercises = () => {
  return prisma.exercise.findMany({
    include: {
      author: true,
+     flaggedBy: true,
+     removedBy: true,
      module: {
        include: { lesson: true }
      }
    }
  })
}
```

Reply:
Yeah, getting the related fields requires a sql JOIN so prisma will not do the extra work unless needed and requested by us.